README.md

```diff
@@ -111,7 +111,7 @@ Output:
 
                               Available formatter          : checkstyle, compact, jslint-xml, json, junit, pretty-error, stylish, table, tap, unix
 
-                              Available formatter for --fix: compats, diff, json, stylish - default: stylish
+                              Available formatter for --fix: compats, diff, json, raw-output, stylish - default: stylish
```

Comment:
Same to https://github.com/textlint/textlint/blob/6497d61122376885e349da9ba98387f74429a0e6/docs/cli.md#L57

Reply:
Updated in https://github.com/textlint/textlint/pull/978/commits/c42d40555d8c6a49de3af71919b0048045ce4920

---

> Naming Issue: I am not sure which is the most obvious.
> 
> - raw-output
> - fixed-output
> - fixed-result

I feel `fixed-result` makes more sense, and I applied it already. Let me know if you have different opinions on this.